src/base/server_type.cpp

```diff
@@ -203,12 +205,18 @@ void finalize_server_type ()
 	  assert (pts_Gl != nullptr);
 	  pts_Gl = nullptr;
 	}
+
       ts_Gl->disconnect_page_server ();
+      if (is_active_transaction_server ())
```

Comment:
finalize log prior sender before  disconnecting the page server; conceptually, after sending receiving the disconnect message/response (`tran_to_page_request::SEND_DISCONNECT_MSG`) no other message can be sent from a transaction server to the page server